hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/buffer/BackupMessagesLoader.java

```diff
@@ -96,19 +103,26 @@ public void loadMessages(MessageRepository messageRepository) {
 
     private void sendMessages(List<BackupMessage> messages) {
         logger.info("Sending {} messages from backup storage.", messages.size());
+        ExecutorService executor = Executors.newFixedThreadPool(16);
```

Comment:
Just a size for beginning, can be tuned in the future, if application start will take too long.